packages/set/src/index.ts

```diff
@@ -18,7 +18,7 @@ const $KEYS = Symbol("track-keys");
 export class ReactiveSet<T> extends Set<T> {
   #triggers = new TriggerCache<T | typeof $KEYS>();
 
-  constructor(values?: readonly T[] | null) {
+  constructor(values?: Iterable<T> | null) {
```

Comment:
c.f. https://github.com/microsoft/TypeScript/blob/main/src/lib/es2015.iterable.d.ts#LL189C21-L189C42

I'm not sure why this is `readonly`; looking at the gitblame for the above line in the TS repo seems to indicate it was never `readonly`.

Reply:
`readonly` was there only to allow passing `readonly` arrays, instead of forcing you to copy to satisfy ts.
with iterable it's not an issue.